installer/package_list.json

```diff
@@ -39,32 +39,30 @@
                 "photon-release",
                 "sed", "grep", "cpio", "gzip", "nano", "db", "libsolv", "libgpg-error", "hawkey", "libassuan", "gpgme", "librepo", "tdnf",
                 "libdnet", "xerces-c", "xml-security-c", "libmspack", "open-vm-tools",
-                "docker",
+                "docker", "rkt",
```

Comment:
Hi Tom,
Thanks for updating rkt.
Can you please revert changes in package_list.json and update the pull request so that we can merge it. We would like to add rkt in minimal at later stage after we are sure that total size of minimal install is under control.

Thanks,
Touseef